arcade/text.py

```diff
@@ -569,6 +569,50 @@ def position(self, point: Point):
             self._label.position = *point, self._label.z
 
 
+class CustomTextSprite(object):
+    def __init__(self, string, Alphabet_Textures, scale=1, 
+                 center_x=0, center_y = 0, 
+                 text_scale=1, text_margin=16, width=100, height = 40,  Background_offset_x=0, Background_offset_y=0, Background_scale=1, Background_Texture=None) -> None:
+        super().__init__()
+        self.Sprite_List = arcade.SpriteList()
```

Comment:
You need to add a type hint to this, as the doc build marks this wrong. Type hints help users know what type an object is.

Reply:
```suggestion
        self.sprite_list = arcade.SpriteList()
```
This should also follow the member variable naming rules eschan145 describer earlier.